api/controllers/autocomplete.js

```diff
@@ -0,0 +1,21 @@
+// const Axios = require('axios');
+const fetch = require('node-fetch');
+const uuid = require('uuid');
+const response = require('../helpers/response');
+
+module.exports = {
+  async autoComplete(req, res) {
+    const { place } = req.params;
+    try {
+      const request = await fetch(
+        `https://maps.googleapis.com/maps/api/place/autocomplete/json?input=${place}
+        &key=AIzaSyCJt2Ze7Xg7mt7mR9ylZp2Nrwhc2AUxu2c&sessiontoken=${uuid()}&geometry`
```

Comment:
is the key not supposed to be a secret???

Reply:
Yeah you are right